QKSMS/src/main/java/com/moez/QKSMS/ui/conversationlist/ConversationListFragment.java

```diff
@@ -62,6 +62,8 @@
     //@Bind(R.id.recording_text) QKTextView recording_text;
 
     private SidebandDBSource sideDb;
+    public static int sTagUpdateFlag;
```

Comment:
I am curious why the scope of this variable static? Are we expecting to access this flag without the ConversationListFragment being initialized or declared?